web/src/user_profile.ts

```diff
@@ -873,7 +873,6 @@ export function show_edit_bot_info_modal(user_id: number, $container: JQuery): v
         // Show the avatar if the user has cleared the image
         $("#bot-edit-form").on("click", ".edit_bot_avatar_clear_button", () => {
             $("#current_bot_avatar_image").show();
-            $(".edit_bot_avatar_file_input").trigger("input");
```

Comment:
I don't know why this line is present here 

this line is causing an issue by unnecessarily triggering the `input` event on the file input field. This, in turn, invokes the function that hides the `#current_bot_avatar_image`. As a result, the avatar image gets hidden unintentionally, even when no actual user input has been made. This behavior is leading to a bug where the image is hidden in scenarios where it should remain visible.